nginx.montagu.conf

```diff
@@ -58,3 +58,20 @@ server {
     server_name _;
     return 301 https://_HOST_$request_uri;
 }
+
+# I don't know how well this will work for the eventual deployment but
+# this approach has the advantage that the jekyll site assumes it is
+# running at the server root (with paths like `/js/...` which don't
+# easily allow running at a path like /static.
```

Comment:
My understanding is that domain names are mapped to IP addresses and can't point to specific ports (see e.g. https://webmasters.stackexchange.com/questions/55123/can-dns-point-to-specific-port). I think the way to achieve what we want is to set up vaccineimpact.org as the server and serve the subdomain montagu.vaccineimpact.org with proxy_pass, as with the other apps. I guess it'll require quite a bit of re-jigging of this config, but my nginx knowledge is very basic. Perhaps @MartinEden can help here?

Reply:
Yeah, my knowledge about these things stops around here.  I figure that this round just proves we can build the docs but I figure some major surgery is still required

Reply:
This looks like it _should_ be possible to this - it actually looks fairly standard perhaps https://www.digitalocean.com/community/tutorials/how-to-set-up-nginx-server-blocks-virtual-hosts-on-ubuntu-16-04

Reply:
That does look pretty straight-forward, want to try it? Could test it the way they suggest by modifying local hosts file... Otherwise we can merge this for testing but may as well remove `vaccineimpact.org` from the server_name as that will never work, and add a comment to say this is just for testing.

Reply:
yeah, I can give this a go.  Can you explain why the server_name bit won't work?  I'm fumbling around with this :)

Reply:
Oh, well it may work when testing with your local hosts file (I don't know!) I just mean that we can't set up DNS for www.vaccineimpact.org to point to our-IP-address:8080. I guess this setup could result in www.vaccineimpact.org:8080 being successfully resolved to our-IP-address:8080, but that's not going to be too helpful. If you change to use server blocks, both listening on port 80, then by all means leave the server_name in.